### PR TITLE
Add PHP 8.0 key signature for carusogabriel

### PR DESCRIPTION
**Pull request by carusogabriel**
Following 0a56570ee64cf9068c52ad1fc8a1616381293be0.

cc @sgolemon @KalleZ

### CODE REVIEW

File: include/gpg-keys.inc [hunk 1]
```diff
@@ -9,6 +9,11 @@ uid                  Kalle Sommer Nielsen (PHP) &lt;kalle@php.net&gt;
 pub   4096R/70D12172 2017-04-14 [expires: 2024-04-21]
       Key fingerprint = 1729 F839 38DA 44E2 7BA0  F4D3 DBDB 3974 70D1 2172
 uid                  Sara Golemon &lt;pollita@php.net&gt;
+
+pub   rsa2048 2020-05-06 [SC] [expires: 2024-05-05]
+      B1C0 C3A5 33D9 9C6F 4C67  1CCE 64E0 AF9F 80E1 80BE
+uid           [ultimate] Gabriel Caruso (Release Manager) <carusogabriel@php.net>
```
Comment (sgolemon) [inline include/gpg-keys.inc:15]: html entify those aligators.
```suggestion
uid           [ultimate] Gabriel Caruso (Release Manager) &lt;carusogabriel@php.net&gt;
```

File: include/gpg-keys.inc [hunk 2]
```diff
@@ -9,6 +9,11 @@ uid                  Kalle Sommer Nielsen (PHP) &lt;kalle@php.net&gt;
 pub   4096R/70D12172 2017-04-14 [expires: 2024-04-21]
       Key fingerprint = 1729 F839 38DA 44E2 7BA0  F4D3 DBDB 3974 70D1 2172
 uid                  Sara Golemon &lt;pollita@php.net&gt;
+
+pub   rsa2048 2020-05-06 [SC] [expires: 2024-05-05]
```
Comment (sgolemon) [inline include/gpg-keys.inc:13]: Minor nit, since this is a new key. I'd go with 4096 bits minimum, but that's your call.

**Comment by sgolemon** [comment]
Once the comments are addressed, you should "merge" this yourself.

((Where "merge" means pick it onto the master branch and push it directly since we don't really "merge" PRs that way))

File: include/gpg-keys.inc (hunk 2 above)
Comment (carusogabriel) [reply include/gpg-keys.inc:13]: I'll do it, thanks for the tip :)

**Comment by carusogabriel** [comment]
> Once the comments are addressed, you should "merge" this yourself.
((Where "merge" means pick it onto the master branch and push it directly since we don't really "merge" PRs that way))

Nice, a similar process that we have in `php-src`.

**Pull request closed without merging**